.github/workflows/govulncheck-action.yml

```diff
@@ -37,17 +35,25 @@ jobs:
         key: ${{ runner.os }}-go-${{ hashFiles ('**/go.sum') }}
         restore-keys: |
           ${{ runner.os }}-go-
-          
+    
+    - name: Test
+      working-directory: govulncheck-action
+      run: |
+        make test
+
+  lint:
+    runs-on: ubuntu-latest
+    name: Lint govulncheck-action
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+    
     - name: Lint
       uses: golangci/golangci-lint-action@v8
       with:
```

Comment:
does the linter run fine without having go installed? Just asking because this is what we do in other repos - we install go and then run the linter - eg: https://github.com/codeready-toolchain/sandboxctl/blob/5569f6cb18cf34bdff47fefcd236ca5cbcd73662/.github/workflows/test.yaml#L44-L63

Reply:
it does, if we refer to [this build job](https://github.com/codeready-toolchain/toolchain-cicd/actions/runs/16030188185/job/45228480330?pr=148)

Reply:
so I guess we could skip the installation of Go in other places, indeed